sys-apps/esh/esh-0.1.1.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Simple templating engine based on shell"
```

Comment:
I can not understand what it does from this DESCRIPTION, you can use up to 80 char. 
perhaps something like `The esh script evaluates shell expressions in plaintext template files.`?

Reply:
Okay, I’ve updated the description.

Reply:
nice